frontend/src/hooks/useFields.tsx

```diff
@@ -418,3 +429,80 @@ export function useFields<T extends Zaak = Zaak>(
     resetFilters,
   ];
 }
+
+const OverflowText = ({
+  text,
+  fieldName,
+}: {
+  text: string;
+  fieldName: string;
+}) => {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [isOverflowing, setIsOverflowing] = useState(false);
+  const dialog = useDialog();
+
+  useEffect(() => {
+    const checkOverflow = () => {
+      if (containerRef.current) {
+        setIsOverflowing(
+          containerRef.current.scrollWidth > containerRef.current.clientWidth,
+        );
+      }
+    };
+
+    const resizeObserver = new ResizeObserver(() => {
+      checkOverflow();
+    });
+
+    if (containerRef.current) {
+      resizeObserver.observe(containerRef.current);
+    }
+
+    checkOverflow();
+
+    return () => {
+      if (containerRef.current) {
+        resizeObserver.unobserve(containerRef.current);
+      }
+    };
+  }, []);
+
+  return (
+    <div style={{ overflow: "hidden" }}>
+      <P
+        ref={containerRef}
+        // We override the textOverflow style, since we implement our own overflow handling.
+        style={{
+          textOverflow: "unset",
+          whiteSpace: "nowrap",
+          overflow: "hidden",
+          width: "calc(100% - 12px)",
+          WebkitMaskImage: isOverflowing
+            ? "linear-gradient(to left, transparent, var(--typography-color-background) 10%)"
+            : "unset",
+          maskImage: isOverflowing
+            ? "linear-gradient(to left, transparent, var(--typography-color-background) 10%)"
+            : "unset",
+        }}
+      >
+        {text}
+      </P>
+      {isOverflowing && (
+        <Button
+          style={{
+            display: "inline-block",
+            position: "absolute",
+            right: -5,
```

Comment:
Use the modern alternatives:

 - https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-end
 - https://developer.mozilla.org/en-US/docs/Web/CSS/inset-block-start